core/common/src/test/java/alluxio/PropertyKeyTest.java

```diff
@@ -58,6 +72,12 @@ public void isValid() throws Exception {
   }
 
   @Test
+  public void aliasIsValid() throws Exception {
+    assertTrue(PropertyKey.isValid(mTestProperty.toString()));
+    assertTrue(PropertyKey.isValid(mNullAliasTestProperty.toString()));
```

Comment:
And, should we consider the situation if the input of the isValid and fromString method is null?
When I run the local test I found if you give the null as input you will get a nullPointerException.